benchmarks/01_trivial/02_soaaos.cpp

```diff
@@ -20,8 +20,23 @@ static Docs docs("soa_to_aos*_stdlib_<type> - test for stdlib implimentation for
 // 256 * sizeof (int) << 12 = 4 Mb - expected to reside in L3.
 #define ARGS4 Arg(256)->Arg(256 << 4)->Arg(256 << 7)->Arg(256 << 12)
 #define ARGS3 Arg(192)->Arg(192 << 4)->Arg(192 << 7)->Arg(192 << 12)
+#define ARGS2 Arg(128)->Arg(128 << 4)->Arg(128 << 7)->Arg(128 << 12)
 
 // Helper functions
+template <typename T> static void init2(T *src, T *dst, int count) {
```

Comment:
Same here, could you please refactor it be a single `init()` function?

Reply:
Done